.golangci.yml

```diff
@@ -0,0 +1,32 @@
+# See: https://golangci-lint.run/usage/linters/
+
+linters:
+  enable:
+    - bodyclose
+    - decorder
+    - dupl
+    - errname
+    - gci
+    - gocritic
+    - gofmt
+    - gosec
+    - govet
+    - grouper
+    - misspell
+    - noctx
+    - stylecheck
+    - tenv
+    - unconvert
+    - wastedassign
+
+linters-settings:
+  stylecheck:
+    checks: ["all", "-ST1003"]
```

Comment:
What is ST1003? Can we leave a comment here?

Reply:
It failed with the following
```
internal/confita/prefixed_env/prefixed_env_test.go:1:1: ST1003: should not use underscores in package names (stylecheck)
package prefixed_env // noling:stylecheck
^
```

I think it would be better not to disable this check but add an exception for `prefixed_env` package. So, I removed these 3 lines from the linter config.